### PR TITLE
feat: grid component added

### PR DESCRIPTION
**Pull request by ShreyashMahajan**
Grid component is added in this
Screenshot (120)
Screenshot (121)
s

### CODE REVIEW

File: componentLibrary.css [hunk 1]
```diff
@@ -666,3 +666,26 @@
 .rating__img {
   height: 2rem;
 }
+
+.grid-two {
+    display: grid;
+    width: 100%;
+    grid-template-columns: repeat(2, 1fr);
```
Comment (sushil-mod) [inline componentLibrary.css:674]: good your using repeat for grid

File: componentLibrary.css [hunk 2]
```diff
@@ -666,3 +666,26 @@
...
+.grid--item {
+     height: 100px;
+     background-color: rgba(211, 37, 25, 0.863);
+}
+
+.grid-three {
+    display: grid;
+    width: 100%;
+    grid-template-columns: repeat(3,1fr);
+    color: black;
```
Comment (sushil-mod) [inline componentLibrary.css]: if possible can use variable

**Review by sushil-mod — APPROVED** [review]
just a suggestion , overall code is good

File: componentLibrary.css (hunk 2 above)
Comment (ShreyashMahajan) [reply componentLibrary.css]: ok i will do that

**Comment by ShreyashMahajan** [comment]
> just a suggestion , overall code is good

thank you

File: component_documentation/componentDoc.css [hunk 3]
```diff
@@ -65,7 +65,7 @@ body {
     position: fixed;
     left: 0;
     width: 20rem;
-    height: 100vh;
+    height: 100vh-67px;
```
Comment (sks-21) [inline component_documentation/componentDoc.css]: You can use calc() for such calculations

**Review by sks-21 — APPROVED** [review]
You have followed a good naming convention. Go ahead and merge.

File: component_documentation/componentDoc.css (hunk 3 above)
Comment (ShreyashMahajan) [reply component_documentation/componentDoc.css]: thank you, i will do that

**Pull request merged**